src/sidebar/SearchView.js

```diff
@@ -36,6 +38,12 @@ class SearchView extends Gtk.ScrolledWindow {
   constructor(params = {}) {
     super(params);
     this.connect("notify::search-term", this.#onNotifySearchTem);
+
+    this._adj = this._search_list_view.get_vadjustment();
```

Comment:
I'm using it in ``#onGestureClick`` to figure out which row was middle clicked on

Reply:
Oh, just to avoid a long ``.`` chain :P and also to make it similar to ``BrowseView`` but it uses ``this._adj`` for other stuff as well. I'll just directly access the property here ig.